src/net/socket_utils.zig

```diff
@@ -10,125 +9,109 @@ const Channel = sig.sync.Channel;
 const Logger = sig.trace.Logger;
 const ExitCondition = sig.sync.ExitCondition;
 
-const UdpSocket = @import("zig-network").Socket;
+const network = @import("zig-network");
+const UdpSocket = network.Socket;
```

Comment:
can we fix the imports in this file -- see contributing.md

Reply:
https://github.com/Syndica/sig/pull/486/commits/767607c3c46beb88cf4533d9bbabba71b9b3e634